content/useStyles: a hook to manage your styles - Agustín Prieto - ES.md

```diff
@@ -0,0 +1,64 @@
+
+![Un hook para gobernarlos a todos](images/useStylesMeme.jpg)
+
+DISCLAIMER: Esto es una referencia; useStyles no fue creado con la intención de gobernar al mundo.
+
+## ¿Qué es useStyles y cómo puede ayudarme?
+
+Organizar los estilos en React Native puede ser complicado, un dolor de cabeza si no aplicas buenas práctias de programación. useStyles viene con la idea de alivianar dicha carga brindando las herramientas para que puedas generar aplicaciones donde los estilos tengan su propio lugar y puedan residir de forma armónica con los componentes.
+
+Si resumimos que hace useStyles: es un hook, el cual te permite consumir de forma eficiente, sencilla y ordenada los estilos de tu aplicación. Por lo que si te atrae la idea de mejorar la forma en que esribes tus estilos y así generar aplicaciones más limpias, te invito a que te pases por el [repositorio](https://github.com/rootstrap/react-native-use-styles) y los pruebes por ti mismo. 
+
+A continuación veremos un ejemplo de uso.
+
+## Uso
+ 
+El siguiente ejemplo muestra como aplicaríamos estilos de forma nativa para generar un simple container con un texto dentro.
+
+https://gist.github.com/agustito37/cca8292de8f3639972af9d9130dc2784
+
+Visto de esta forma, no parece un mal código, ¿verdad? Ahora qué pasa cuando el componente empieza a crecer. 
+
+El componente unas semanas después, unas muy malas semanas después:
+https://gist.github.com/agustito37/d89db60561fbe5073e340f3b6c6c35b4
+
+Como verán, nuestro lindo componente empezó a ser devorado lentamente por nuestros feos estilos. Si bien hay muchas formas de mejorar este código, dado que React Native no tiene una forma clara y sencilla de manejar estilos, estas malas prácticas son comunes.
+
+Ahora implementemos lo mismo pero con useStyles:
+https://gist.github.com/agustito37/2f49201ea6f7a7cfce3318181f6e00c4
+
+Perdonen :cry:. Mis ojos se llenaron de alegría al ver un código organizado. Sniff. Capaz esto les hace acordar un poco al uso de CSS; si bien useStyles está inspirado en dicho concepto; esta es una solución creada especialmente para React Native, poco tiene que ver con CSS. 
+
+Volviendo al previo ejemplo, se puede ver como el hook `useStyles` devuelve una función, que es utilizada luego para consumir nuestros estilos. Ya habrán notado algunas declaraciones peculiares, como un hook al que se le pasan dependencias, y algunos prefijos como `.`, `&`, `$` y `:`. A continuación, cuando veamos la deficinión de los estilos, vamos a ver que significan.
+
+Vemos los estilos globales:
+https://gist.github.com/agustito37/de4c35b5dbfab9127080948d0c07ae5f
+
+Y por último, veamos los estilos del componente:
+https://gist.github.com/agustito37/2759b5e981d2810ceb77bdfd36a1851a
+
+useStyles te probee las interfaces `GlobalStyles` y `Styles` para definir estilos globales y locales.
```

Comment:
```suggestion
useStyles te probee las interfaces `GlobalStyles` y `Styles` para definir estilos globales y locales.

Estilos globales:
https://gist.github.com/agustito37/de4c35b5dbfab9127080948d0c07ae5f

Estilos del componente:
https://gist.github.com/agustito37/2759b5e981d2810ceb77bdfd36a1851a
```

Maybe something like this? Putting first the sentence saying the options and then expanding on them

Reply:
Yes, it has more sense hihi